Add app to customJS object (global app is deprecated)

ebullient opened the pull request:
Add the plugin app reference to window.customJS, as the global app reference has been deprecated.

saml-dev commented:
Will the access to `this.app` be similarly deprecated down the road? I'm wondering if this is just delaying the inevitable (which is fine too lol)

saml-dev commented on main.ts:
```diff
@@ -37,6 +37,7 @@ export default class CustomJS extends Plugin {
     };
     this.app.workspace.onLayoutReady(async () => {
       await this.loadClasses();
+      window.customJS.app = this.app;
```
I think it makes more sense to add this down in `loadClasses` where we initialize the customJS object.

ebullient commented:
> Will the access to `this.app` be similarly deprecated down the road? I'm wondering if this is just delaying the inevitable (which is fine too lol)

I don't think so. The App is passed to the plugin as part of the API for setting up plugins (there are a lot of reasons we have to have it).

Fixed, rebased, repushed

saml-dev commented:
Thanks! Will cut a release with this later today.

saml-dev commented:
Released in 1.0.16
